### PR TITLE
feat(test-tool): add test tool sample bot app

### PR DESCRIPTION
**Pull request by qinezh**
Add test tool sample bot app for Test Tool evaluation. Note: this sample is not going to onboard to sample gallery.

### CODE REVIEW

File: test-tool-sample-app/.webappignore [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.webappignore
+.fx
+.deployment
+.localConfigs
```
Comment (a1exwang) [inline test-tool-sample-app/.webappignore:4]:
```suggestion
.localConfigs.testTool
.localConfigs
.notification.localstore.json
.notification.testtoolstore.json
```

File: test-tool-sample-app/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "author": "Microsoft",
+    "license": "MIT",
+    "main": "./lib/index.js",
+    "engines": {
+        "node": "16 || 18"
+    },
+    "scripts": {
+        "dev:teamsfx": "env-cmd --silent -f .localConfigs npm run dev",
+        "dev:teamsfx:testtool": "env-cmd --silent -f .localConfigs.testTool npm run dev",
+        "dev:teamsfx:launch-testtool": "env-cmd --silent -f .localConfigs.testTool teamsapptester start",
```
Comment (a1exwang) [inline test-tool-sample-app/package.json:14]:
```suggestion
        "dev:teamsfx:launch-testtool": "env-cmd --silent -f .env.testTool teamsapptester start",
```

File: test-tool-sample-app/.webappignore (hunk 1 above)
Comment (qinezh) [reply test-tool-sample-app/.webappignore:5]: updated.

File: test-tool-sample-app/package.json (hunk 2 above)
Comment (qinezh) [reply test-tool-sample-app/package.json]: updated.

**Review by a1exwang — DISMISSED** [review]

File: test-tool-sample-app/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Teams App Test Tool Sample App
+
+This sample app comprehensively demonstrates the features supported by the Teams App Test Tool, which is a good starting point to try Teams App Test Tool.
+
```
Comment (MuyangAmigo) [inline test-tool-sample-app/README.md:4]: The sample app includes common development scenarios when building Teams Bot applications: 
- Send proactive notifications
- Trigger a chat command and receive a response
- Trigger an Adaptive Card actions

**Review by a1exwang — APPROVED** [review]

**Pull request merged**